lib/tasks/api.rake

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+namespace :api do
+  desc "Sets all data services for the api"
+  task up: :environment do
+    fail "Seed tasks can only be used in the development enviroment" if Rails.env.production?
+
+    STDOUT.puts "Your are trying to recreate all indexes and the database. Are you sure? (y/n)"
+    input = STDIN.gets.strip
+    if input == "y"
+      Rake::Task["elasticsearch:create_all_indexes"].invoke
+      Rake::Task["db:create"].invoke
+      Rake::Task["db:schema:load"].invoke
+      Rake::Task["db:seed:development:base"].invoke
+    else
+      puts "So sorry for the confusion"
+    end
+  end
+
+  desc "removes all data services for the api"
+  task delete_all_data: :environment do
```

Comment:
since all the indexes/tables are connected its a good idea to have a request to clear everything up. That way when one creates the whole setup one can be sure there are not loose ends. As we discussed in the dev meeting all the methods inside are standard rake tasks that can be triggered anyway. As mentioned there, the risk is the same.